src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -50,7 +51,7 @@ public MovieSessionResponseDto add(@RequestBody MovieSessionRequestDto requestDt
     public List<MovieSessionResponseDto> findAvailableSessions(@RequestParam Long movieId,
                                                 @RequestParam
             @DateTimeFormat(pattern = DateTimePatternUtil.DATE_PATTERN)
-                                                        LocalDate date) {
+                                                       @Valid LocalDate date) {
```

Comment:
```suggestion
                                                       LocalDate date) {
```
this parameter redundant